### PR TITLE
Transform CreationManager into a reusable factory class

### PR DESCRIPTION
**Pull request by taubenangriff**

### CODE REVIEW

**Comment by taubenangriff** [comment]
working PR for modinfo creator

**Comment by taubenangriff** [comment]
Added DLCs to the mix. 

This should be it for now, rest comes later

**Comment by jakobharder** [comment]
Removing Singleton, Manager etc. is a good move. 👍

The `Factory` might confuse people with the factory pattern, but that's fine.

You seem to like this modinfo creator feature :P

File: ModManager_Classes/Utils/ModinfoCreationManager.cs [hunk 1]
```diff
@@ -43,41 +46,76 @@ public String ModDepsJoined
         }
         private String _mod_deps_joined;
 
-        public static ModinfoCreationManager Instance { get; } = new ModinfoCreationManager();
+        public ObservableCollection<DlcId> DLCs { get; set; } = new();
 
-        public ModinfoCreationManager()
+        public ModinfoFactory()
```
Comment (jakobharder) [inline ModManager_Classes/Utils/ModinfoCreationManager.cs:51]: I'd handle all GitHub actions warnings. VS should also show them all the time.

They only distract from important warnings.

**Review by jakobharder — APPROVED** [review]

**Pull request merged**